[Bug]: Fix install components

### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0
Shadcn Vue version: 1.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```


### Link to minimal reproduction

https://www.shadcn-vue.com/docs/components/input.html

### Steps to reproduce

when i try to install https://www.shadcn-vue.com/docs/components/input.html

### Describe the bug

when i am try to install button, input any other any other components it got this error


"npx shadcn-vue@latest add input
Need to install the following packages:
  shadcn-vue@0.1.5
Ok to proceed? (y) y
node:internal/modules/cjs/loader:1048
  const err = new Error(message);
              ^

Error: Cannot find module 'detype'
Require stack:
- C:\Users\91983\AppData\Local\npm-cache\_npx\360157ef5dc9b5fb\node_modules\shadcn-vue\dist\index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at file:///C:/Users/91983/AppData/Local/npm-cache/_npx/360157ef5dc9b5fb/node_modules/shadcn-vue/dist/index.js:121:21
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:308:24)
    at async loadESM (node:internal/process/esm_loader:42:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\91983\\AppData\\Local\\npm-cache\\_npx\\360157ef5dc9b5fb\\node_modules\\shadcn-vue\\dist\\index.js'
  ]
}

Node.js v20.6.1"


### Expected behavior

how to solve this issue ?

### Conext & Screenshots (if applicable)


![error](https://github.com/radix-vue/shadcn-vue/assets/72514035/aecf38e9-3628-49df-be99-697969c74890)

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -52,6 +52,7 @@
     "chalk": "5.3.0",
     "commander": "^11.0.0",
     "cosmiconfig": "^8.3.6",
+    "detype": "^0.6.3",
     "diff": "^5.1.0",
     "execa": "^8.0.1",
     "fs-extra": "^11.1.1",
@@ -76,7 +77,6 @@
     "@types/lodash.template": "^4.5.1",
     "@types/prompts": "^2.4.4",
     "@vitest/ui": "^0.34.4",
-    "detype": "^0.6.3",
     "tsup": "^7.2.0",
     "type-fest": "^4.3.1",
     "typescript": "^5.2.2"

